components/tool-navbar.vue

```diff
@@ -22,6 +22,7 @@
                     <b-nav-item
                         v-for="(navItem, _key) in navItems"
                         :active="pageName === navItem.fullName"
+                        :data-cy="'menu-item-' + navItem.pageName"
```

Comment:
As per recommended best practice, any attribute accessed as an identifier by cypress should be separate from identifiers used by the app itself in order to avoid brittleness.

Our current standard is `data-cy` for the attribute name.

Reply:
Yes, I considered whether to select by ID value or UI label too – if I'm understanding your comment correctly.

Whatever we do should be standard in our testing implementation, but my intuition is not to rely on the label text but rather than the html attribute value. Arguably, it is less likely for the attribute value to change.